syntax = "proto3";

package orbax_model_jd_assets_map;

option java_multiple_files = true;

// Defines a map from original asset paths to their new locations within the
// saved model. This is used by the Jax Data to correctly locate and
// load assets.
message JDAssetsMap {
  // Key: The original path (or name) of an asset.
  // Value: The relative path of the asset within the Orbax saved model folder.
  map<string, string> assets = 1;
}
